docs/contributors/waku-fleets.md

```diff
@@ -20,8 +20,7 @@ and configured to bridge toy-chat messages to the `#waku channel` on the Vac Dis
 
 ### Fleet deployment rationale
 
-The `test` fleet is automatically updated after every commit to the `nim-waku` `master` branch
-and is therefore the most up to date representation of Waku v2 development.
+The `test` fleet is automatically updated after every commit to the `nim-waku` repository `master` branch and is therefore the most up to date representation of Waku v2 development.
```

Comment:
```suggestion
The `test` fleet is automatically updated after every commit to the `nwaku` repository `master` branch and is therefore the most up to date representation of Waku v2 development.
```

Reply:
Yeah, nim-waku is all over the place. I would say we can run the *.md files through `sed 's/nim-waku/nwaku/g` in a follow up PR?